docs/community/com-ingest/com-ingest-source-kafka-static.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Kafka Static Ingest Reference
+layout: default
+parent: Import data
+grand_parent: Community
+---
+
+## Kafka Static Ingester
+
+The Kafka Static ingester reads JSON messages from Kafka and writes the data to FeatureBase. It uses a static schema file which is also written in JSON. 
+
+## Usage
```

Comment:
This section runs into the problem of where these parameters are actually added, found, modified, etc.